### ISSUE TITLE
BUG: Blog Page not Mirroring Other Pages Based on .toml

### ISSUE THREAD

**Issue by HanselPalencia** [pacollins/hugo-future-imperfect-slim#185]
# Bug Report

## Describe the bug

Blog page not mirroring the home, about, and contact page/sidebar

## To Reproduce

1. Download clean version of theme
2. Delete the Filler Blog Posts

## Expected behavior

When changing the .toml the specifications should go across all pages (home, blog, about, etc.)

## Screenshots

https://hpblogging.netlify.app/
You can see here to what I am referring to 

## Environment

 - Not sure what my hugo version is.
 - Chrome 83.0.4103.116
 - Laptop Windows
 - https://github.com/HanselPalencia/Blog

## Additional context

I think deleting all of the blog posts at once might have caused my /content/blog/ folder to delete itself. This in turn might have caused issues that are leading to the bug.

Basically this could be a really dumb question, but any help would be appreciated.
